check_chromium_src_config.json5

```diff
@@ -0,0 +1,130 @@
+        ],
+        "chrome/browser/bookmarks/android/bookmark_bridge.cc": [
+            "BraveBookmarkBridge"
+        ],
+        "chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.cc": [
+            "CHROME_BROWSER_WEB_APPLICATIONS_COMMANDS_CLEAR_BROWSING_DATA_COMMAND_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_UTILS_H_"
+        ],
+        "chrome/browser/chrome_content_browser_client.cc": [
+            "CHROME_BROWSER_WEB_APPLICATIONS_POLICY_WEB_APP_POLICY_MANAGER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_SYSTEM_WEB_APPS_SYSTEM_WEB_APP_MANAGER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_REGISTRAR_H_"
+        ],
+        "chrome/browser/metrics/google_update_metrics_provider_win.cc": [
+            "BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION"
+        ],
+        "chrome/browser/prefs/browser_prefs.cc": [
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_"
+        ],
+        "chrome/browser/sync/chrome_sync_client.cc": [
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_SYNC_BRIDGE_H_"
+        ],
+        "chrome/browser/ui/webui/chrome_web_ui_controller_factory.cc": [
+            "CHROME_BROWSER_WEB_APPLICATIONS_SYSTEM_WEB_APPS_SYSTEM_WEB_APP_MANAGER_H_",
+            "CHROME_BROWSER_WEB_APPLICATIONS_WEB_APP_PROVIDER_H_"
+        ],
+        "chrome/browser/ui/webui/favicon_source.cc": [
+            "BRAVE_CHROMIUM_SRC_EXTENSIONS_COMMON_CONSTANTS_H_",
+            "CHROME_BROWSER_SEARCH_INSTANT_SERVICE_H_",
+            "EXTENSIONS_BROWSER_EXTENSION_REGISTRY_H_",
+            "EXTENSIONS_COMMON_CONSTANTS_H_",
+            "EXTENSIONS_COMMON_MANIFEST_H_"
+        ],
+        "chrome/install_static/install_constants.h": [
+            "BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION"
+        ],
+        "chrome/install_static/install_util.cc": [
+            "BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION"
+        ],
+        "chrome/installer/mini_installer/appid.h": [
+            "CHROME_INSTALLER_MINI_INSTALLER_APPID_H_"
+        ],
+        "chrome/installer/setup/install_worker.cc": [
+            "BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION"
+        ],
+        "chrome/installer/setup/user_experiment.cc": [
+            "BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION"
+        ],
+        "components/permissions/permission_request_data.h": [
+            "BRAVE_CHROMIUM_SRC_COMPONENTS_PERMISSIONS_PERMISSION_REQUEST_DATA_PERMISSION_CONTEXT_BASE"
+        ],
+        "sandbox/win/src/interceptors_64.h": [
+            "TargetGetModuleFileNameExA64",
+            "TargetGetModuleFileNameExW64"
+        ],
+        "v8/src/builtins/builtins-utils.h": [
+            "BUILTIN"
+        ]
+    }
+}
```

Comment:
new line at end

Reply:
Fixed in dde637633ee8265bb9dbc7a7b39cc41c4c46d9a1